src/niftyone/figures/generator.py

```diff
@@ -65,12 +53,20 @@ def create_generators(config: dict[str, Any]) -> list["ViewGenerator"]:
     """Create selected generators dynamically from config with default settings."""
     generators: list["ViewGenerator"] = []
 
-    for group in config.values():
-        query = group.get("query", "")
-        views = group.get("views", [])
-
-        for view in views:
-            generators.append(create_generator(view=view, query=query))
+    for group in config.get("figures", None).values():
+        queries = group.get("queries", "")
```

Comment:
I don't think the default here is something we need to worry about. It may probably better to just have it raise / not generate anything if none is provided since users will have different data organization / modalities /etc. Plus we do ship a default config with it currently that it reads from (maybe we shouldn't).